index.js

```diff
@@ -11,5 +11,5 @@ app.listen(PORT, () => {
 })
 
 app.get('/', (req, res) => {
```

Comment:
@DonaldBazile you should specify the path for `portfolio` here (e.g. `/portfolio` instead of `/`) and keep `/` for `home.html`